runbotics/runbotics-scheduler/src/utils/page/page.ts

```diff
@@ -25,9 +25,9 @@ export const getPage = async <T>(repo: Repository<T>, options: FindManyOptions<T
         first: options.skip === 0,
         totalElements: count,
         totalPages,
-        last: totalPages === 0 || page + 1 === totalPages,
+        last: page === totalPages - 1,
```

Comment:
when there are no processes in the db `page === 0` and `totalPages === 0`, so your update will result in `last: false` when there are no processes